### DISCUSSION TITLE
Welcome to Accelerated Text Discussions!

### DISCUSSION THREAD

**Discussion by zmedelis** [accelerated-text/accelerated-text#320] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

We're just getting started with the Discussions board, it seems like a great potential support channel and a place where we can help with Accelerated Text usage questions and discuss ideas.

Questions, Ideas, and Suggestions are always welcome!

**Comment by Kilumile** [comment]
Thank you
